ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__metal-4.18-latest-dataplane-x86.yaml

```diff
@@ -0,0 +1,84 @@
+build_root:
```

Comment:
Instead of creating a new config file (**_openshift-eng-ocp-qe-perfscale-ci-main__metal-4.18-latest-dataplane-x86.yaml_**), would you consider using the existing telco one? https://github.com/openshift/release/blob/master/ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__metal-telco-x86.yaml